services/web-app/components/catalog/catalog.js

```diff
@@ -348,6 +348,14 @@ function Catalog({ collection, data, type, filter: defaultFilter = {}, glob = {}
     setPossibleFilterValues(getFilters({ collection, type }))
   }, [collection, type])
 
+  useEffect(() => {
+    const resultsCount = filteredAssets.length
+    const maxPageNumber = Math.max(Math.ceil(resultsCount / pageSize), 1)
+    if (page > maxPageNumber) {
```

Comment:
Might just want to add one more check if page isn't an integer, set it to 1. E.g. if you manually put `&page=asdf` in the URL.

Reply:
I didn't add it because in that case, useQueryState will dismiss the invalid value after calling the supplied validation function (line 219) 
<img width="417" alt="image" src="https://user-images.githubusercontent.com/40550942/160669830-7ac640d9-6194-46f0-a16b-5c045e40c627.png">
so it will return the last valid value, by the time it gets here the input has already been cleared.
Do you still want me to add a check for redundancy?

Reply:
The page held up just fine with `&page=asdf`, so we're good! Thanks for explaining how `useQueryState` is handling this.

Reply:
There's a few edge cases here for things that can be coerced to numbers. All of these will pass the `!isNaN` check, but might not be returning something useful:

```js
!isNaN(false)
!isNaN(null)
!isNaN(true)
!isNaN(1.123e9)
!isNaN('')
```

`parseInt` might be a better validation option, since I'm not sure if `parseNumbers: true` tries both `parseInt` and `parseFloat`.

Reply:
Done!